CGPs/cgp-0184.md

```diff
@@ -0,0 +1,189 @@
+---
+cgp: 184
```

Comment:
Please change to 185 as it clashes with the mento draft pr submitted prior to this pr

Reply:
done